packages/web-app-admin-settings/src/views/Spaces.vue

```diff
@@ -120,7 +121,7 @@ export default defineComponent({
         'name asc',
         'driveType eq project'
       )
-      const drives = drivesResponse.map((space) =>
+      const drives = drivesResponse.map((space: Drive) =>
```

Comment:
why does this need an explicit type? If drivesResponse is properly typed, this should not be necessary